src/Rest.php

```diff
@@ -69,10 +69,13 @@ private function execute($method, $action, $request = null)
             $status      = (int) curl_getinfo($ch, CURLINFO_HTTP_CODE);
             $header_size = curl_getinfo($ch, CURLINFO_HEADER_SIZE);
             $headers     = $this->parseHeaders(substr($data, 0, $header_size));
-            if ($status === 400 &&
-                                (int) $headers['X-RateLimit-Remaining'] === 0) {
+            if (
+              $status === 400 &&
+              (int) $headers['x-ratelimit-remaining'] === 0) {
```

Comment:
it's more readable if the brakets would be both in a new line

Reply:
updated